operator/v1/types_network.go

```diff
@@ -411,6 +411,16 @@ type OVNKubernetesConfig struct {
 	// gatewayConfig holds the configuration for node gateway options.
 	// +optional
 	GatewayConfig *GatewayConfig `json:"gatewayConfig,omitempty"`
+	// v4InternalSubnet is the v4 join subnet to be used by ovn-kubernetes in case the
+	// default one is being already used by something else.
+	// Default is 100.64.0.0/16
```

Comment:
"join subnet" is meaningless to the user. "is a v4 subnet used internally by ovn-kubernetes".

"Must not overlap with any other subnet being used by OpenShift or by the node network"

Needs a hint about how big it needs to be. (Does it depend on the cluster size? Number of nodes? Number of pods?)

Can this be changed after install or does it need to be set at install time?

Reply:
fixed.